lib/engine/game/g_system18/map_ms_customization.rb

```diff
@@ -245,14 +245,18 @@ def map_ms_init_round
         end
 
         def map_ms_reorder_players
-          return unless @players.any? { |p| !p.companies.empty? }
+          # call base game method if players have no privates
+          return self.class.superclass.instance_method(:reorder_players).bind_call(self) unless @players.any? do |p|
```

Comment:
Hmmm I don't love needing `bind_call` but it's of course tricky with the metaprogramming already in place.

IMO this is at least a little easier to understand, take it or leave it:

```suggestion
          return Engine::Game::Base.instance_method(:reorder_players).bind_call(self) unless @players.any? do |p|
```

Reply:
Yeah, it's probably better to reference Game explicitly.